.cirrus.yml

```diff
@@ -161,6 +158,7 @@ test_freebsd_task:
 
   check_reproducible_script: |
     if [ -n "$CHECK_REPRODUCIBLE" ]; then
+      taskset 1
```

Comment:
You need to write `taskset 1` that as the prefix of the actual command, so `taskset 1 gmake check_reproducible` - but if that does 2 builds, you only want to apply it to 1 of them.

Reply:
Yup, thanks! The error message was very clear too. Unfortunately no taskset on macos, so I am shooting on the dark in the CI. :)

Reply:
Sweet! I was able to reproduce the failure you got (not the same, but a similar one) with `taskset 1`. Now I brought the fix from master, let's see how it goes!